apps/aeutils/priv/epoch_config_schema.json

```diff
@@ -111,6 +111,23 @@
                     "type" : "boolean",
                     "default": true,
                 },
+                "resolver_max_retries": {
+                    "description" : "Maximum number of retries at resolving the host name of untrusted peers.",
+                    "type" : "integer",
+                    "minimum" : 0,
+                    "default" : 7
+                },
+                "resolver_backoff_times" : {
+                    "description" : "Waiting time intervals (milliseconds) before each retry at resolving the host name of peers (both trusted and untrusted).",
```

Comment:
Maybe we should add that if there is less values than `resolve_max_retries`, the last one is reused...

Reply:
You are right. Let me add that.

Reply:
Addressed in 77a68efe